### PR TITLE
feat: added alert

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created Simple and icon variants for 5 types of alerts:

1. A primary alert
2. A secondary alert
3. A success alert
4. A warning alert
5. A danger alert

simple-alert

icon-alert

### CODE REVIEW

**Review by shuntzhere — COMMENTED** [review]
Suggested a minor change. Otherwise looks good.

File: stylesheets/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.alert {
+    max-width: 35rem;
+    padding: var(--padding-one);
+    margin: var(--margin-one);
+    border-radius: 0.25rem;
+    color: var(--gray-50);
+}
+
+
+/* Simple Alerts */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+.warning-alert {
+    background-color: var(--yellow-500);
+}
+
+.danger-alert {
+    background-color: var(--red-500);
+}
+
+
+/* Alerts with icons */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+/* Primary alert with icon */
+
+.primary-icon-alert {
+    color: var(--primary-color);
+    border: 1px solid var(--primary-color);
+}
+
+/* Secondary alert with icon */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+.secondary-icon-alert {
+    color: var(--secondary-color);
+    border: 1px solid var(--secondary-color);
+}
+
+/* Success alert with icon */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+.success-icon-alert {
+    color: var(--green-500);
+    border: 1px solid var(--green-500);
+}
+
+/* Warning alert with icon */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+.warning-icon-alert {
+    color: var(--yellow-500);
+    border: 1px solid var(--yellow-500);
+}
+
+/* Danger alert with icon */
```
Comment (ImrozNezami) [inline stylesheets/components/alert.css]: remove comment

File: stylesheets/components/alert.css (hunk 1 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

File: stylesheets/components/alert.css (hunk 2 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

File: stylesheets/components/alert.css (hunk 3 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

File: stylesheets/components/alert.css (hunk 4 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

File: stylesheets/components/alert.css (hunk 5 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

File: stylesheets/components/alert.css (hunk 6 above)
Comment (nakulsharma15) [reply stylesheets/components/alert.css]: Ok, removed!

**Review by shuntzhere — APPROVED** [review]

**Pull request merged**